script/foundry/utils/DeployHelper.sol

```diff
@@ -546,12 +662,12 @@ contract DeployHelper is Script, BroadcastManager, JsonDeploymentHandler, Storag
 
     /// @dev get the salt for the contract deployment with CREATE3
     function _getSalt(string memory name) private view returns (bytes32 salt) {
-        salt = keccak256(abi.encode(name, block.number));
+        salt = keccak256(abi.encode(name, create3SaltSeed));
```

Comment:
Informational only - with this change we will need a way to keep track of the previous salt seeds used before running the script

Reply:
Agreed. The deployment would fail if we deploy contracts again with the same seed on the same chain.